schema/featureTable.schema.json

```diff
@@ -4,19 +4,23 @@
     "title" : "Feature Table",
     "type" : "object",
     "description" : "A set of semantics containing per-tile and per-feature values defining the position and appearance properties for features in a tile.",
-    "patternProperties" : {
-        ".*" : {
+    "additionalProperties" : {
+        "anyOf": [
```

Comment:
Are you sure about this?  The Feature Table properties could be anything: strings, numbers, arrays, objects with their schema, etc.  It is up to the **tile format** to limit it.